includes/elements/column.php

```diff
@@ -919,6 +919,7 @@ protected function content_template() {
 		?>
 		<div class="elementor-<?php echo esc_attr( $wrapper_element ); ?>-wrap">
 			<div class="elementor-background-overlay"></div>
+			<div class="elementor-edit-click-area"></div>
```

Comment:
```suggestion
			<div class="e-edit-click-area"></div>
```
Maybe? (also others)

Reply:
Good idea, renamed them all